src/cgroups/v2/systemd_manager.rs

```diff
@@ -45,6 +46,19 @@ struct CgroupsPath {
 
 impl SystemDCGroupManager {
     pub fn new(root_path: PathBuf, cgroups_path: PathBuf) -> Result<Self> {
+        // TODO: create the systemd unit using a dbus client.
+        let cgroups_path = Self::new_cgroups_path(cgroups_path)?;
+        let cgroups_path = Self::get_cgroups_path(cgroups_path)?;
```

Comment:
@Furisto This API is slightly confusing i.e. `new_cgroups_path` vs. `get_cgroups_path`. I realize naming is hard but perhaps we can come up with a better name here?

The corresponding method is called `initPath` in [runc](https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/systemd/v2.go#L354-L376) - perhaps we call it `init_path` or simply `generate_path`? :shrug: 

@utam0k  WDYT?

Reply:
Was not super happy with the naming either. Reworked it. Let me know what you think.

Reply:
Very nice, Much better.